src/measure_theory/integral/riesz_markov_kakutani.lean

```diff
@@ -109,3 +112,140 @@ begin
 end
 
 end riesz_subadditive
+
+section urysohn_bounded_nnreal
+
+lemma real.isometry_const_sub (c : ℝ) :
+  isometry (λ (r : ℝ), c - r) :=
+begin
+  intros x y,
+  simp_rw [edist_dist],
+  apply congr_arg ennreal.of_real,
+  simp only [real.dist_eq, sub_sub_sub_cancel_left, abs_sub_comm],
+end
```

Comment:
Sorry for being MIA for so long! I'm ready to open the new PR but just wanted to check in with you first, is it alright to put everything needed for the nnreal-valued Urysohn (division/subtraction in a seminormed_comm_group is isometry, subtraction in nnreal is Lipschitz) into one PR or should each lemma get a seperate PR, since they're all going into different files?

Reply:
I think a single PR is fine.